django_project/base/templates/project/new_detail.html

```diff
@@ -93,7 +93,7 @@ <h4><img src="{% thumbnail sponsor.sponsorship_level.logo 20x20 %}"> {{ sponsor.
                                 <img class="img-rounded tooltip-toggle"
                                      {% thumbnail sponsor.sponsor.logo 50x50 as logo %}
                                         {% if logo %}
-                                            src="{{ logo }}"
+                                            src="{% thumbnail sponsor.sponsor.logo 50x50 %}"
```

Comment:
@sumandari it would be better if you created a template tag for these lines of code, as they repeat often. But I guess it's fine for now.

Reply:
Thanks @dimasciput, I will submit another PR for the template tag.